contracts/nouns/nouns.sol

```diff
@@ -85,53 +67,27 @@ contract Nouns {
     // DKG
     //
 
-    // Final summed polynomial coefficients.
-    // After round 1, PK[0] = final committee PK.
-    uint[2][] public PK_coeffs;
-    uint[2][] public PK_shares;
-
-    // Round 1
-
-    mapping(uint => uint[2][]) round1_C_coeffs;
-    mapping(address => bool) public round1_done;
-    uint round1_received;
-
-    // Round 2
-    mapping(uint => mapping(uint => bool)) public round2_shares_received;
-    uint round2_num_shares;
-    // mapping(uint => mapping(uint => uint)) public enc_sk_shares;
-    // mapping(uint => mapping(uint => uint[2])) public enc_eph_PK;
+    uint public tally_threshold;
 
     // Lookup table for vote counts
     mapping(uint => mapping(uint => uint)) public lookup_table;
 
-    // Events
-    event Round1Complete();
-    event Round2Share(uint indexed recip_id, uint sender_id, uint enc_sk_share, uint[2] enc_eph_PK);
     event TallyComplete(/*uint indexed vote_id, */ uint yay, uint nay, uint abstain);
 
     // Generator Point
     uint public constant Gx = 5299619240641551281634865583518297030282874472190772894086521144482721001553;
     uint public constant Gy = 16950150798460657717958625567821834550301663161624707787222815936182638968203;
 
     constructor(
+        address dkg_address, // DKG contract
```

Comment:
let's stay consistent with the leading `_` convention for the argument here.

Not sure if this is the best convention, and it's not used consistently everywhere (at some point we should fix a convention and choose a linter #19) but let's keep things locally consistent for now.

Reply:
Fixed